lib/widgets/components/user_icon.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:coffeecard/widgets/components/images/coffee_image.dart';
```

Comment:
Since we now have two components related to the user, should they not be in a folder for themselves?